digit/server.go

```diff
@@ -0,0 +1,36 @@
+package digit
```

Comment:
I like splitting everything into a smaller package. This allows us to import it and also means it reduces the likelihood that we have any namespace collisions